src/actions/read-rule.js

```diff
@@ -15,11 +15,13 @@ module.exports = ({ rule, docs, friendlyDocPath }, name) => {
   const fixable = rule.meta.fixable
   const { description, extraDescription, recommended } = rule.meta.docs
 
-  const nl = detectNewline(docs)
+  const heading = `# ${description} (${name})`
 
-  const newDocs = [`# ${description} (${name})`]
-    .concat(docs.split(nl).slice(1))
-    .join(nl)
+  const regTest = /^\s*#[^\r\n]*(\r?\n)?/
```

Comment:
Can you rename this `headingRe`?

Reply:
Or something else that makes more sense to you, as long as it clarifies that it’s matching the heading.